plugin.program.chrome.launcher/resources/language/English/strings.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="utf-8" standalone="yes"?>
+<strings>
+  <string id="30001">Add Website</string>
```

Comment:
just converted all the stuff into po's then .. not sure if the english translation should be as it is now ..